setup.cfg

```diff
@@ -3,7 +3,7 @@ VCS=git
 style=pep440-post
 versionfile_source=src/orion/core/_version.py
 versionfile_build=orion/core/_version.py
-tag_prefix=
+tag_prefix=v
```

Comment:
What is the difference in the convention here?

Reply:
I don't remember exactly the source of the problem, but from what I remember it's about conda which is looking for only X.Y.Z format and fails if we provide vX.Y.Z format without explicitly telling it in the tag_prefix. 
@abergeron Am I wrong?

Reply:
The difference is that the previous config is wrong since you actually use the `v` prefix for tags.  It has not been a problem before because pip "fixes" the version and strips the v.  My packaging pipeline is less flexible and includes the v.